packages/react/ds-core/src/ui/Card/styles.css

```diff
@@ -0,0 +1,43 @@
+/* @canonical/generator-ds 0.10.0-experimental.2 */
+
+.ds.card {
+  /* prevent overflow of child margins */
+  overflow: auto;
+
+  box-shadow: var(--intent-box-shadow, var(--card-box-shadow));
+  color: var(--intent-color-text, var(--card-color-text));
+  background-color: var(--intent-color, var(--card-color-background));
+  border-width: var(--card-border-width);
+  border-style: var(--card-border-style);
+  border-color: var(--card-color-border);
+  margin-block-end: var(--card-margin-bottom);
+  margin-inline-end: var(--card-margin-left);
+  padding-block: var(--card-padding-vertical);
+  padding-inline: var(--card-padding-horizontal);
+
+  /* Content bleed mode - remove padding from card, add to inner elements */
+  &.content-bleed {
+    padding: 0;
+
+    & .card-inner {
+      overflow: auto;
+      padding-block: var(--card-inner-padding-vertical);
+      padding-inline: var(--card-inner-padding-horizontal);
+    }
+  }
+
+  & .content {
+    & > *:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  /*
+        In VF, the thumbnail was spaced away from its following content using a utility class that added extra margin.
+        This allows us to more semantically add that spacing without using a global utility class.
+        TODO is this a good way of doing this? If so, how should the token setup for this work?
+    */
+  & .card-thumbnail-container {
+    margin-bottom: var(--spacing-vertical-medium);
+  }
```

Comment:
Highlighting a point of interest here: I was able to provisionally eliminate the need for a [margin-adjusting utility class](https://github.com/canonical/vanilla-framework/blob/6af35433368f82d76d8deb7effebca47b434ee76/templates/docs/examples/patterns/card/header.html#L9) by applying margin to a neighboring element using a container class. This will need some further validation with other components, but it may be a decent way for us to get around some utility classes.